test/integration/targets/copy/tasks/main.yml

```diff
@@ -169,8 +169,9 @@
 # Test copy with mode=preserve
 - name: Set file perms to an odd value
   file:
-    path: '{{ output_file }}'
+    path: '{{ role_path }}/files/foo.txt'
```

Comment:
workaround: copy the file to another directory on the controller side and then do the copy to the remote machine from there.